docs/master/api-reference/directives.md

```diff
@@ -2892,32 +2892,49 @@ enum Operator {
 
 ### Usage
 
+The most basic way to use this directive is with the dynamic `WhereConstraints` input and the `@whereConstraints` directive with no arguments.
+
 ```graphql
+type Person {
```

Comment:
I like adding that. While reworking a lot of `@whereConstraints`, i included this in https://github.com/nuwave/lighthouse/pull/1099.